back/src/pods/storage/api/storage.api.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  UserSessionContext,
+  UserSession,
+  ConnectSessionInfo,
+  VotesFromRooms,
+} from 'dals/user';
+import { roomApi } from 'pods/room';
+
+//TODO implement queries
+
+// This is just a demo approach, storing in memory session Info
```

Comment:
I would update this comment to 

```ts
// Only to be used on a local development environment (in case you don't want run the mongo db version of the API, e.g. no docker installed locally)
```

Reply:
Oops make no sense the comment here, maybe i the mock file (here I guess we should remove it)